src/SDKs/_metadata/sql_resource-manager.txt

```diff
@@ -1,33 +1,33 @@
 ﻿Installing AutoRest version: latest
 
-> autorest@2.0.4280 preinstall C:\Users\yaiyun\AppData\Roaming\npm\node_modules\autorest
+> autorest@2.0.4280 preinstall C:\Users\nivithla\AppData\Roaming\npm\node_modules\autorest
 > node ./preinstall-check
 
-C:\Users\yaiyun\AppData\Roaming\npm\autorest -> C:\Users\yaiyun\AppData\Roaming\npm\node_modules\autorest\dist\app.js
+C:\Users\nivithla\AppData\Roaming\npm\autorest -> C:\Users\nivithla\AppData\Roaming\npm\node_modules\autorest\dist\app.js
 + autorest@2.0.4280
-updated 1 package in 2.315s
+updated 1 package in 1.234s
 
 AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=C:\code\azure-sdk-for-net\src\SDKs
-AutoRest code generation utility [version: 2.0.4280; node: v8.11.1]
+cmd.exe /c autorest.cmd https://github.com/nivimsft/azure-rest-api-specs/blob/uploadCustomerTdeCertificate/specification/sql/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=D:\WS\AzureSDKForNetMyFork\src\SDKs
+AutoRest code generation utility [version: 2.0.4280; node: v8.11.0]
 (C) 2018 Microsoft Corporation.
 https://aka.ms/autorest
-   Loading AutoRest core      'C:\Users\yaiyun\.autorest\@microsoft.azure_autorest-core@2.0.4280\node_modules\@microsoft.azure\autorest-core\dist' (2.0.4280)
+   Loading AutoRest core      'C:\Users\nivithla\.autorest\@microsoft.azure_autorest-core@2.0.4280\node_modules\@microsoft.azure\autorest-core\dist' (2.0.4280)
    Loading AutoRest extension '@microsoft.azure/autorest.csharp' (~2.2.51->2.2.57)
    Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.43->2.3.43)
 
-2018-06-14 07:14:29 UTC
+2018-06-18 18:25:28 UTC
 1) azure-rest-api-specs repository information
-GitHub fork: Azure
-Branch:      master
-Commit:      69592a62dd01225f16a9b398664c3b9d6ddc8268
+GitHub fork: nivimsft
```

Comment:
You will need to generate from Azure/master before this PR can be merged

Reply:
Sounds good! Waiting for PR to be merged